modules/concept-docs/pages/data-model.adoc

```diff
@@ -10,8 +10,29 @@ Several data structures are supported by the SDK, including map, list, queue, an
 
 include::{version-common}@sdk:shared:partial$data-model.adoc[tag=intro]
 
-include::{version-common}@sdk:shared:partial$data-model.adoc[tag=structures]
 
+== Data Structures 
+
+Data structures in Couchbase are similar in concept to data structures in Scala:
+
+* *Map* is an iterable over a set of keys and values, like a https://docs.scala-lang.org/overviews/collections-2.13/maps.html[Scala Map].
+* *List* is is a finite immutable sequence.
```

Comment:
It's Buffer in Scala, and is mutable.   It extends Scala's mutable.AbstractBuffer